src/zampy/datasets/land_cover.py

```diff
@@ -0,0 +1,254 @@
+
+    def convert(
+        self,
+        ingest_dir: Path,
+        convention: str | Path,
+    ) -> bool:
+        converter.check_convention(convention)
+        ingest_folder = ingest_dir / self.name
+
+        data_file_pattern = "land-cover_LCCS_MAP_*.nc"
+
+        data_files = list(ingest_folder.glob(data_file_pattern))
+
+        for file in data_files:
+            # start conversion process
+            print(f"Start processing file `{file.name}`.")
+            ds = xr.open_dataset(file)
+            ds = converter.convert(ds, dataset=self, convention=convention)
+            # TODO: support derived variables
+            # TODO: other calculations
+            # call ds.compute()
+
+        return True
+
+
+def unzip_raw_to_netcdf(
+    ingest_folder: Path,
+    file: Path,
+    overwrite: bool = False,
+) -> None:
+    """Convert a downloaded zip netcdf file to a standard CF/Zampy netCDF file.
+
+    Args:
+        ingest_folder: Folder where the files have to be written to.
+        file: Path to the land cover .zip archive.
+        overwrite: Overwrite all existing files. If False, file that already exist will
+            be skipped.
+    """
+    ncfile = ingest_folder / file.with_suffix(".nc").name
+    if ncfile.exists() and not overwrite:
+        print(f"File '{ncfile.name}' already exists, skipping...")
+    else:
+        ds = extract_netcdf_to_zampy(file)
+        ds.to_netcdf(path=ncfile)
+
+
+def extract_netcdf_to_zampy(file: Path) -> xr.Dataset:
+    """Extract zipped data and convert to zampy format.
+
+    Since the original land cover field is too large to fit into
+    the memory in general, in this function the loaded land cover
+    data are regridded. They are regrid to a resoltuion of 0.25
+    degree, same as the native resolution of ERA5 data.
+
+    Args:
+        file: Path to the land cover .zip archive.
+
+    Returns:
+        Coarse land cover data in zampy format.
+    """
+    with TemporaryDirectory() as temp_dir:
+        unzip_folder = Path(temp_dir)
+        with ZipFile(file, "r") as zip_object:
+            zipped_file_name = zip_object.namelist()[0]
+            zip_object.extract(zipped_file_name, path=unzip_folder)
+
+        # only keep land cover class variable
+        with xr.open_dataset(unzip_folder / zipped_file_name) as ds:
+            var_list = [var for var in ds.data_vars]
+            raw_variable = "lccs_class"
+            var_list.remove(raw_variable)
+            ds = ds.drop_vars(var_list)  # noqa: PLW2901
+
+            # coarsen to fit into memory
+            ds = ds.sortby(["lat", "lon"])  # noqa: PLW2901
+            ds = ds.rename({"lat": "latitude", "lon": "longitude"})  # noqa: PLW2901
+            new_grid = xarray_regrid.Grid(
+                north=90,
+                east=180,
+                south=-90,
+                west=-180,
+                resolution_lat=0.25,  # same as resolution of ERA5
+                resolution_lon=0.25,
```

Comment:
I think this could be a higher default resolution. We should preferably be able to expose this in some way to the user, but I'm not sure how :/ 

```suggestion
                resolution_lat=0.05,
                resolution_lon=0.05,
```

Reply:
Indeed, it would be nice for the user to know it. But I have no idea where to put the information...Anyway, I will change the values.

Reply:
Can you make an issue out of it so we can look at it later, and it's not a forgotten part of the code? :)

Reply:
There you go #41 .